topics/computational-chemistry/tutorials/covid19-docking/tutorial.md

```diff
@@ -0,0 +1,342 @@
+>        - *"Generate 3D coordinates"*: `Yes`
+> 2. Rename the output file `Enumerated candidates SDF`.
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > The **Compound conversion** {% icon tool %} tool is based on the OpenBabel toolkit {% cite OBoyle2011 %}.
+>    {: .comment}
+>
+{: .hands_on}
+
+## Splitting the SD-file into a collection
+
+The next stage is to split the SD-file with the candidate ligands into a set of smaller SD-files.
+
+
+> ### {% icon question %} Questions
+>
+> Why is splitting the file necessary?
+>
+> > ### {% icon solution %} Solution
+> >
+> > The rDock tool performs one docking at a time (more technically: the task is not parallelized, as it uses only a single CPU). Therefore, splitting the large SD-file into many small files allows the work to be carried out by multiple Galaxy jobs in parallel, so it completes faster.
+> >
+> > In the original study, this kind of parallelization was absolutely essential because of the enormous dataset; at some points, there were 5,000 docking jobs running concurrently on the European Galaxy server. Even on the much smaller scale of this tutorial, we can speed things up considerably using this trick.
+> >
+> {: .solution}
+{: .question}
+
+> ### {% icon hands_on %} Hands-on: Split the SD-files
+>
+> 1. **Split file** {% icon tool %} with the following parameters:
+>    - *"Select the file type to split"*: `SD-files`
+>    - *"SD-file to split"*: `Enumerated candidates SDF`
+>        - *"Specify number of output files or number of records per file?"*: `Number of output files`
+>            - *"Number of new files"*: `10`
+>        - *"Method to allocate records to new files"*: `Alternate output files`
+>
+>
+{: .hands_on}
+
+***TODO***: *Consider adding a question to test the learners understanding of the previous exercise*
+
+
+# Active site preparation
+
+The active site also needs to be prepared for docking, using the following steps: 1) conversion to MOL2 format, and 2) generation of the active site using the **rbcavity** {% icon tool %} tool.
+
+> ### {% icon details %} Differences with the original study
+>
+> This stage was carried out as described here. However, it was repeated for each of the fragment hit structures, not just the `Mpro-x0195_0_apo-desolv.pdb` file used here. See [here](https://covid19.galaxyproject.org/cheminformatics/2-ActiveSitePrep/) for more details.
+{: .details}
+
+
+## Convert protein structure to MOL2 format
+
+The receptor file we are using is in PDB format, but the rDock tool we use for docking requires an input in MOL2 format. Therefore, we first convert the file.
+
+> ### {% icon hands_on %} Hands-on: Conversion to MOL2 format
+>
+> 1. **Compound conversion** {% icon tool %} with the following parameters:
+>    - *"Molecular input file"*: `Receptor PDB` dataset.
+>    - *"Output format"*: `Sybyl Mol2 format (mol2)`
+> 2. Rename the output file `Receptor MOL2`.
+>
+>
+{: .hands_on}
+
+## Generate active site definition
```

Comment:
maybe move this one to the active-site-preparation?

Reply:
it is: 'Active site preparation' is a level 1 heading and 'Generate active site definition' is level 2.

Is this unclear?